src/applications/check-in/components/pages/Edit/Relationship.jsx

```diff
@@ -59,33 +56,33 @@ export default function Relationship(props) {
 
   const relationshipOptions = useMemo(
     () => ({
-      SPOUSE: 'Spouse',
-      CHILD: 'Child',
-      BROTHER: 'Brother',
-      SISTER: 'Sister',
-      MOTHER: 'Mother',
-      FATHER: 'Father',
+      SPOUSE: t('spouse'),
+      CHILD: t('child'),
+      BROTHER: t('brother'),
+      SISTER: t('sister'),
+      MOTHER: t('mother'),
+      FATHER: t('father'),
```

Comment:
I just merged an update to this that changes these values. Probably need to do a rebase and fix them up again.